### PR TITLE
Initial setup for Polyglot 2016

### PR DESCRIPTION
**Pull request by saem**
First round of updates for the 2016 conference.

### CODE REVIEW

File: _harp/public/_shared/sponsors.ejs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="container">
     <h1>Sponsors</h1>
 
-    <div class="sponsors">
+    <!-- <div class="sponsors">
```
Comment (chrisnicola) [inline _harp/public/_shared/sponsors.ejs]: Is that... commented code...? ping: @tavisrudd /codereview

Comment (tavisrudd) [inline _harp/public/_shared/sponsors.ejs]: Yeah, that should work.

**Comment by tavisrudd** [comment]
@chrisnicola can you add @saem and me as members of the github polyglot organization?

**Comment by tavisrudd** [comment]
Looks good to me as starting point.  @chrisnicola can we add pics from the pro photographer last year to a repo here or an s3 bucket somewhere?

**Comment by chrisnicola** [comment]
@tavisrudd remind me who organized the photographer last year? But yes I'll make sure we get this up.

**Comment by saem** [comment]
I think it might have been a WealthBar employee.

**Comment by tavisrudd** [comment]
@chrisnicola Saem and I don't have permissions to merge PRs on this repo. Can you please add us as admin members of https://github.com/PolyglotConf?

**Comment by tavisrudd** [comment]
@chrisnicola nvm. I didn't see the outstanding invite.

**Pull request merged**